platform/vcs-impl/src/com/intellij/openapi/vcs/update/AbstractCommonUpdateAction.java

```diff
@@ -301,7 +301,7 @@ private static boolean supportingVcsesAreEmpty(final ProjectLevelVcsManager vcsM
     private LocalHistoryAction myLocalHistoryAction;
 
     public Updater(final Project project, final FilePath[] roots, final Map<AbstractVcs, Collection<FilePath>> vcsToVirtualFiles) {
-      super(project, getTemplatePresentation().getText(), true, VcsConfiguration.getInstance(project).getUpdateOption());
+      super(project, getTemplatePresentation().getText(), true, ActionInfoEx.maybeOverrideBackGroundOption(myActionInfo, VcsConfiguration.getInstance(project).getUpdateOption()));
```

Comment:
Why do you want to override the background option here? Do you want the Update Project operation to be always-modal?